customer-bill/src/main/java/org/fiware/tmforum/customer_bill/rest/CustomerBillOnDemandApiController.java

```diff
@@ -0,0 +1,91 @@
+package org.fiware.tmforum.customer_bill.rest;
+
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.annotation.Controller;
+import io.reactivex.Single;
+import io.reactivex.schedulers.Schedulers;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.fiware.customer_bill.api.CustomerBillOnDemandApi;
+import org.fiware.customer_bill.model.CustomerBillOnDemandCreateVO;
+import org.fiware.customer_bill.model.CustomerBillOnDemandVO;
+import org.fiware.tmforum.common.exception.NonExistentReferenceException;
+import org.fiware.tmforum.common.validation.ReferenceValidationService;
+import org.fiware.tmforum.customer_bill.TMForumMapper;
+import org.fiware.tmforum.customer_bill.domain.customer_bill.CustomerBillOnDemand;
+import org.fiware.tmforum.customer_bill.exception.CustomerBillOnDemandCreationException;
+import org.fiware.tmforum.customer_bill.repository.CustomerBillOnDemandRepository;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+@Slf4j
```

Comment:
For (easy) enabling the conformance tests, please make the controller base path configurable. See(e.g. copy;)) from IndividualApiController